src/traccuracy/utils.py

```diff
@@ -0,0 +1,37 @@
+import copy
+
+import numpy as np
+
+from traccuracy import NodeFlag, TrackingGraph
+from traccuracy.matchers import Matched
```

Comment:
I'm not sure about the typing issue, but for the docs problem can you try switching the imports like below and see if that fixes the problem?
```suggestion
from traccuracy._tracking_graph import NodeFlag, TrackingGraph
from traccuracy.matchers._base import Matched
```